Bug Report: some system requirements can not be set for win32_app

<!-- Please search existing issues to avoid creating duplicates. -->

## Description

When trying to use the the following settings an error message pops up, preventing apply the correct settings in Intune.

**Resource**: microsoft365_graph_beta_device_and_app_management_win32_app 
**Parameters**:
- minimum_cpu_speed_in_mhz
- minimum_free_disk_space_in_mb
- minimum_memory_in_mb
- minimum_number_of_processors

### Steps to Reproduce

Configure a microsoft365_graph_beta_device_and_app_management_win32_app with the parameters minimum_cpu_speed_in_mhz, minimum_free_disk_space_in_mb, minimum_memory_in_mb or minimum_number_of_processors.

Example:
``` terraform
resource "microsoft365_graph_beta_device_and_app_management_win32_app" "notepad_v8_9_1" {
  allow_available_uninstall = true

  app_installer = {
    installer_file_path_source = "C:\\temp\\Software\\Output\\npp.8.9.1.Installer.x64.intunewin"
  }

  // Optional
  app_icon = {
    icon_url_source = "https://upload.wikimedia.org/wikipedia/commons/f/f5/Notepad_plus_plus.png"
  }

  // Optional
  categories = [
    "Business",
    "Productivity",
  ]

  description     = "Notepad++ v8.9.1 DES"
  publisher       = "Don Ho Pub"
  developer       = "Don Ho Dev"
  display_name    = "Notepad++ v8.9.1 DIS"
  display_version = "8.9.1"
  file_name       = "npp.8.9.1.Installer.x64.intunewin"
  information_url = "https://notepad-plus-plus.org/"

  owner = "Jan"
  notes = "Notes"

  allowed_architectures             = ["x64"]
  minimum_supported_windows_release = "Windows11_23H2"

  minimum_free_disk_space_in_mb = "2000"

  install_experience = {
    device_restart_behavior = "allow"
    max_run_time_in_minutes = 60
    run_as_account          = "system"
  }

  setup_file_path        = "Invoke-AppDeployToolkit.exe"
  install_command_line   = "Invoke-AppDeployToolkit.exe -DeploymentType Install"
  uninstall_command_line = "Invoke-AppDeployToolkit.exe -DeploymentType Uninstall"

  # Detection Rules
  rules = [
    {
      rule_type                  = "detection"
      rule_sub_type              = "file_system"
      path                       = "C:\\Program Files\\Notepad++"
      file_or_folder_name        = "notepad++.exe"
      check_32_bit_on_64_system  = false
      file_system_operation_type = "version"
      lob_app_rule_operator      = "greaterThanOrEqual"
      comparison_value           = "8.9.1"
    },
  ]

  return_codes = [
    {
      return_code = 0
      type        = "success"
    },
    {
      return_code = 1707
      type        = "success"
    },
    {
      return_code = 3010
      type        = "softReboot"
    },
    {
      return_code = 1641
      type        = "hardReboot"
    },
    {
      return_code = 1618
      type        = "retry"
    }
  ]

}
```

### Expected Behavior

settings to be applied in Intune.


### Actual Behavior

terraform runs into an error when trying to apply this settings

### Environment

- **OS**: Windows
- **Terraform Version**: Tested with Terraform 1.14.8 and OpenTofu 1.11.6
- **Provider Version**: 0.54.0-alpha
- **Other dependencies**: no

### Additional Context

Add any other context about the problem here. Include logs, error messages, or links to related issues if available.


### Logs & Console Output

```
2026-06-14T14:43:31.490+0200 [ERROR] vertex "microsoft365_graph_beta_device_and_app_management_win32_app.notepad_v8_9_1" error: Provider produced inconsistent result after apply
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to microsoft365_graph_beta_device_and_app_management_win32_app.notepad_v8_9_1, provider "provider[\"registry.opentofu.org/deploymenttheory/microsoft365\"]" produced an unexpected new value: .minimum_free_disk_space_in_mb: was cty.NumberIntVal(2000), but now null.
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
```



### Priority

How critical is this issue? Choose one of the following:
- Medium (Needs to be fixed but not urgent)

--- a/internal/services/resources/device_and_app_management/graph_beta/win32_app/construct.go
+++ b/internal/services/resources/device_and_app_management/graph_beta/win32_app/construct.go
@@ -35,6 +35,7 @@ func constructResource(ctx context.Context, data *Win32LobAppResourceModel, inst
 	convert.FrameworkToGraphString(data.PrivacyInformationUrl, requestBody.SetPrivacyInformationUrl)
 	convert.FrameworkToGraphString(data.Notes, requestBody.SetNotes)
 	convert.FrameworkToGraphString(data.Owner, requestBody.SetOwner)
+	convert.FrameworkToGraphInt32(data.MinimumFreeDiskSpaceInMB, requestBody.SetMinimumFreeDiskSpaceInMB)
 	convert.FrameworkToGraphString(data.MinimumSupportedWindowsRelease, requestBody.SetMinimumSupportedWindowsRelease)
 	convert.FrameworkToGraphBool(data.AllowAvailableUninstall, requestBody.SetAllowAvailableUninstall)
 	convert.FrameworkToGraphBool(data.IsFeatured, requestBody.SetIsFeatured)

